scipy/stats/tests/test_morestats.py

```diff
@@ -37,6 +37,24 @@
 
 
 class TestBayes_mvs(TestCase):
+    def test_basic(self):
+        data = [6, 9, 12, 7, 8, 8, 13]
+        mean, var, std = stats.bayes_mvs(data)
+        assert_almost_equal(mean.statistic, 9.0)
+        assert_allclose(mean.minmax, (7.1036502226125329, 10.896349777387467),
```

Comment:
I took it directly from the docstring example. Not sure what was the original source but I can verify by hand at least.